test/blackbox-tests/test-cases/pkg/build-pkg-deps.t

```diff
@@ -0,0 +1,74 @@
+This test verifies the @pkg-deps alias fetch and build the project dependencies
+without building the project itself.
+
+  $ . ./helpers.sh
+
+Create a fake library to install in the target project as a dependency:
```

Comment:
I didn't really get why you need a fake library and executable in this test. I see the following as sufficient:

1. Write a lock directory
2. Build your new alias
3. Verify that the sources exist

Everything else seems redundant.

Reply:
As we want to build the library and ensure it is installed, you need the fake library to make sure dune build correctly. Otherwise, it will fail when you try to use it as a dependency.